hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/LogEventViewGenerator.java

```diff
@@ -43,6 +47,19 @@
     }
   }
 
+  private String getSummary(Attributes attributes) {
+    if (null == attributes
```

Comment:
nit: The same checks lines `51-55` are used in other function `convertAttributes`, so should we move this check to function `isEmptyAttributes()`, and use both the places

```
private boolean isEmpty(Attributes attributes) {
if (null == attributes
        || null == attributes.getAttributeMap()
        || attributes.getAttributeMap().isEmpty()) {
      return null;
    }
}